conf/.htaccess.security

```diff
@@ -7,5 +7,8 @@ RedirectMatch 403 /wp-includes/.+\.php$
   RewriteRule ^login\.php$ /wp-login.php [QSA,END]
   RewriteRule ^wp-login\.php$ - [NS,F,END]
   RewriteRule ^xmlrpc\.php$ - [F,END]
+
+  #Ensure .git folder is not accessible
+  RewriteRule \.git/.*?$ - [F,NC]
```

Comment:
I don't understand what you're matching after the directory.
```suggestion
  RewriteRule \.git/ - [F,END]
```
Note that this also applies (and should also apply) to other .git folders in the filesystem; e.g., `wp-content/plugins/core-standards/.git/HEAD`